src/Compilers/Core/Portable/Symbols/ITupleTypeSymbol.cs

```diff
@@ -14,7 +13,7 @@ namespace Microsoft.CodeAnalysis
     /// </remarks>
     public interface ITupleTypeSymbol : ITypeSymbol
```

Comment:
I'm ok with that.  *however* with the caveat that we add some methods (possibly extension) to INamedTypeSymbol to allow you to ask what the constituent set of tuple element types and tuple element names are.